crates/integration-tests/src/tests/filtering.rs

```diff
@@ -66,24 +66,25 @@ async fn filters_by_comparisons_on_elements_of_array_field() -> anyhow::Result<(
     Ok(())
 }
 
-#[tokio::test]
-async fn filters_by_comparisons_on_elements_of_array_of_scalars() -> anyhow::Result<()> {
-    assert_yaml_snapshot!(
-        graphql_query(
-            r#"
-            query MyQuery {
-              movies(where: { cast: { _eq: "Albert Austin" } }) {
-                title
-                cast
-              }
-            }
-            "#
-        )
-        .run()
-        .await?
-    );
-    Ok(())
-}
+// This doesn't seem to work anymore in the latest engine version
```

Comment:
This can be re-enabled if you put the `CompatibilityConfig` date back I believe?

Reply:
I tried setting `CompatibilityConfig` to an earlier date, and it didn't help so I'm removing this test. This functionality isn't supported until the next ndc-spec version anyway.